File: 프로그래머스/0/120848. 팩토리얼/팩토리얼.py
def solution(n):
    answer = 1
    facNum = 1
    
    while facNum < n :        
        if facNum >= n :
            print(answer)
            break  
        else :
            answer += 1    
            facNum *= answer   
    
    if facNum != n :
        answer -= 1
        
    return answer

File: 프로그래머스/0/120848. 팩토리얼/팩토리얼.js
function solution(n) {
        var answer = 1;
    let facNum = 1;
    while (facNum < n) {        
        answer += 1;
        facNum *= answer;
    }
    
    if(facNum != n){
        answer -= 1;
    }
    return answer;
}